lib/modules/dosomething/dosomething_api/resources/kudos_resource.inc

```diff
@@ -189,15 +189,13 @@ function _kudos_resource_index($reportbackitems, $count) {
  */
 function _kudos_resource_create($reportback_item_id, $term_ids, $northstar_id) {
   if (variable_get('rogue_collection', FALSE)) {
-    if (isset($northstar_id)) {
-      $northstar_user->id = $northstar_id;
-    } else {
+    if (!isset($northstar_id)) {
```

Comment:
Can we set `$northstar_id` within this conditional so we don't have to do the ternary below?